fcrepo-http-api/src/main/java/org/fcrepo/http/api/ContentExposingResource.java

```diff
@@ -141,6 +146,10 @@
 
     protected abstract String externalPath();
 
+    protected Deskolemizer deskolemizer() {
+        return deskolemizer == null ? deskolemizer = new Deskolemizer(translator(), null) : deskolemizer;
```

Comment:
That would be great if we could encode subjective measures of "readability" into checkstyles. I try not to encroach on other's style preferences, but when a style impedes personal readability, I raise the flag.
Thanks for the niggly update.